.github/workflows/codeception.yml

```diff
@@ -51,7 +51,13 @@ jobs:
         run: |
           cd "/tmp/wordpress/src/wp-content/plugins/${GITHUB_REPOSITORY#*/}"
           pwd
-          php vendor/bin/codecept run
+          php vendor/bin/codecept run wpunit Data/ExporterTest.php
```

Comment:
Looks like this got a lot more complicated.... Can we do this in one line instead of one per file?

Reply:
Hey @stephywells before I set it to run all, but it would always return an error `Warning: mysqli_query(): Couldn't fetch mysqli` . I looked around and found this https://githubmemory.com/repo/lucatume/wp-browser/issues/497 . They will work well when singular and not in parallel

Reply:
Fixed it, works now as a single suite

Reply:
Error for MySQL was when trying to run all different tests at once (acceptance and wpunit)

Reply:
This is looking much better now. As long as you are positive a failed test will fail here in Github, we can go ahead and merge this.

Reply:
Yes, I'm positive. I have some failed tests locally for testing purposes and the results fail like if its phpunit